src/components/Header.tsx

```diff
@@ -5,7 +5,7 @@ import truncateEthAddress from 'truncate-eth-address'
 import { WalletContext } from '../context/WalletContext'
 
 const style = {
-  wrapper: "p-4 w-screen flex justify-between items-center",
+  wrapper: "p-4 w-full flex justify-between items-center",
```

Comment:
don't commit any styles, let Roberto do it.

Reply:
Done